src/Spout/Common/Helper/GlobalFunctionsHelper.php

```diff
@@ -248,7 +248,7 @@ public function is_readable($fileName)
      */
     public function basename($path, $suffix = null)
```

Comment:
looking at the method signature (https://www.php.net/manual/en/function.basename), it should be:
```
public function basename($path, $suffix = '')
```
Changing the signature (and associated PHPDoc) would be a better fix I believe.